app/src/main/java/edu/uc/reedws/musiclink/ui/main/ApplicationViewModel.kt

```diff
@@ -17,16 +17,16 @@ class ApplicationViewModel(application: Application): AndroidViewModel(applicati
         fetchPlaylists()
     }
 
-    fun fetchPlaylists() {
-        viewModelScope.launch {
-            playlists = _playlistService.fetchPlaylists()
-        }
-    }
-
```

Comment:
I moved the fetchPlaylists() down in the code because I learned from one of the professor videos (Sprint 1 code review) that private functions should be down below all the public functions in the class as it is one of the Kotlin coding conventions.